### PR TITLE
Fix notification ArgsProps typing

### LINKED ISSUE #7832: Wrong typing of notification component ArgsProps type
### Version
2.13.4

### Environment
independent of environment

### Reproduction link
[https://codepen.io/anon/pen/KXoBmG?editors=001](https://codepen.io/anon/pen/KXoBmG?editors=001)

### Steps to reproduce
using the `notification` component in a typescript project and try to set the `style` property

### What is expected?
Type of property `style` should be `object`

### What is actually happening?
Type of property `style` is `string` which leads to compiler errors => one can not set the style currently.

### PR DESCRIPTION
**Pull request by bzums**
close #7832

First of all, thank you for your contribution! :-)

Please makes sure that these checkboxes are checked before submitting your PR, thank you!

* [x] Make sure that you propose PR to right branch: bugfix for `master`, feature for latest active branch `feature-x.x`.
* [x] Make sure that you follow antd's [code convention](https://github.com/ant-design/ant-design/wiki/Code-convention-for-antd).
* [x] Run `npm run lint` and fix those errors before submitting in order to keep consistent code style.
* [x] Rebase before creating a PR to keep commit history clear.
* [x] Add some descriptions and refer relative issues for you PR.

Extra checklist:

**if** *isBugFix* **:**

  * [ ] Make sure that you add at least one unit test for the bug which you had fixed.

**elif** *isNewFeature* **:**

  * [ ] Update API docs for the component.
  * [ ] Update/Add demo to demonstrate new feature.
  * [ ] Update TypeScript definition for the component.
  * [ ] Add unit tests for the feature.

### CODE REVIEW

**Comment by bzums** [comment]
I cannot add a unit test here since the tests seem to be written in javascript-only (correct me if I am wrong). Since it is just a typing issue, we cannot test it if we do not use types in the tests.

**Comment by bzums** [comment]
I also suggest to use the more strict (and in this case accurate) `object` type (https://blog.mariusschulz.com/2017/02/24/typescript-2-2-the-object-type) here instead of `Object`. I adapted the documentation, therefore.

File: components/notification/index.tsx [hunk 1]
```diff
@@ -101,7 +101,7 @@ export interface ArgsProps {
   duration?: number;
   icon?: React.ReactNode;
   placement?: NotificationPlacement;
-  style?: string;
+  style?: object;
```
Comment (afc163) [inline components/notification/index.tsx]: React.CSSProperties

https://github.com/ant-design/ant-design/blob/9a5894c46ee9fe48644af72da59cc70c83c285ec/components/button/button.tsx#L51

Comment (bzums) [reply components/notification/index.tsx]: Ah, I see, will use that, thanks!

Comment (bzums) [reply components/notification/index.tsx]: @afc163 one question: how to reference this type in the documentation?

Comment (afc163) [reply components/notification/index.tsx]: [React.CSSProperties](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/e434515761b36830c3e58a970abf5186f005adac/types/react/index.d.ts#L794) would be nice.

`[React.CSSProperties](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/e434515761b36830c3e58a970abf5186f005adac/types/react/index.d.ts#L794)`

**Pull request merged**